src/analysis/retail/hunter/marksmanship/modules/talents/AimedShot.tsx

```diff
@@ -62,6 +62,13 @@ class AimedShot extends Analyzer {
     );
   }
 
+  //Pin Cushion reduces the cooldown of Aimed Shot by 2 seconds
+  onSteadyShot(event: CastEvent) {
+    if (this.selectedCombatant.hasTalent(TALENTS_HUNTER.PIN_CUSHION_TALENT)) {
+      this.spellUsable.reduceCooldown(TALENTS_HUNTER.AIMED_SHOT_TALENT.id, 2000, event.timestamp);
```

Comment:
the final field `event.timestamp` is optional and doesn't need to be specified when using the event's timestamp (that's the default)

Reply:
removed.